.github/workflows/ci.yml

```diff
@@ -26,22 +26,22 @@ jobs:
 
       # Caching
       - name: Cache Cargo Registry
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           path: ~/.cargo/registry
           key: ${{ runner.os }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
       - name: Cache Cargo Bin
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           path: ~/.cargo/bin
           key: ${{ runner.os }}-cargo-bin-${{ hashFiles('**/Cargo.lock') }}
       - name: Cache Cargo Git
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           path: ~/.cargo/git
           key: ${{ runner.os }}-cargo-git-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
Since v2 allows multi-path caching, can you combine these into one single cache?

Reply:
Yep, you're right!